.buildkite/ftr_platform_stateful_configs.yml

```diff
@@ -13,6 +13,7 @@ disabled:
   - x-pack/test/fleet_api_integration/config.base.ts
   - x-pack/test/functional_basic/apps/ml/config.base.ts
   - x-pack/test/functional_basic/apps/transform/config.base.ts
+  - x-pack/platform/test/api_integration_basic/config.basic_license.ts
```

Comment:
to use as a default config for an extension in "platform" and "solutions" directories